src/node_dotenv.cc

```diff
@@ -161,10 +175,10 @@ void Dotenv::ParseLine(const std::string_view line) {
     // Remove any leading/trailing spaces from unquoted values.
     while (!value.empty() && std::isspace(value.front())) value.erase(0, 1);
     while (!value.empty() && std::isspace(value.back()))
-      value.erase(value.size() - 1);
+      value.erase(
+          value.size() - 1);
```

Comment:
This change seems to be unnecessary.

Reply:
formatter. I can revert but I think my formatter was trying to get stuff under 80 lines or make it look "more readable"